SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.3 BLOCK -->
```

Comment:
Do we need this to exist as a markdown document or can it be a policy in the repositories security policies (link at the top)?

Reply:
I'd prefer not to link to compliance-related documentation as it tends to churn quite a bit and likely to be 404 before we know it.  If CELA requires to update down the road we'll be informed through some other compliance process.